api/Makefile

```diff
@@ -12,4 +12,3 @@ build::
 	cp ../README.md ../LICENSE bin/
 
 lint::
-	tslint -c ../tslint.json -p tsconfig.json
```

Comment:
Does the linter complain about all the deprecation?

Reply:
it's more that i overwrote existing comments, which had linting-preferences specified in them.  i don't want to have to go add those things back in.  so just easier (imo) to disable linting entirely.

Reply:
Ahh, makes sense. I think that should be fine given the plans here.